examples/multi_resources.yaml

```diff
@@ -0,0 +1,26 @@
+name: multi-resources
+
+resources:
+  cloud: gcp
+  # zone: us-east-1d
+  use_spot: True
+
+  region: [us-west1, asia-east1]
+
+  # spot_recovery: FAILOVER
+  spot_recovery: EAGER_NEXT_REGION
+
+  # Ordered Accelerators List
+  resources_ordered: True
+  accelerators: ['A100-40GB:1', 'V100:1', 'T4:1', 'K80:1']
```

Comment:
We probably need to check the semantic of `sky exec` when multiple resources are specified.